paper/paper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Game Theory and Python: An educational tutorial to game theory and
+repeated games using Python"
+tags:
+- game theory
+- repeated games
+- prisoner's dilemma
+- axelrod python
+authors:
+- name: Nikoleta E. Glynatsi
+- name: Dr Vincent A. Knight
+  ORCID: 
+  affiliation: 1 affiliations:
+- name: Department of Mathematics, Cardiff University, Senghennydd Rd, Cardiff CF24 4AG
+  index: 1, 2
+date: some date in 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+Game Theory and Python is an open source educational tutorial aimed to
```

Comment:
`These materials are an open source educational tutorial aimed at introducing participants to game theory [...]`